spec/bitpagos/client_spec.rb

```diff
@@ -25,6 +26,17 @@
       end
     end
 
+    context "API key provided is invalid" do
+      let(:api_key) { "OMGSARASA" }
+      let(:exception) { RestClient::Unauthorized }
```

Comment:
@mauro-oto We should make this a gem-specific exception. Maybe something like `Bitpagos::Unauthorized`? 

The user of the gem doesn't need to know that we are using `RestClient` and it will make it easier if the user is already using `RestClient`